.github/CODEOWNERS

```
@@ -1,5 +1,8 @@
 # Translations
 po/da.po @mgeisler
 po/de.po @ronaldfw @fechu
+po/el.po @Mandragorian
+po/fr.po @sakex
```

```suggestion
po/fr.po @sakex @lb034582341
```

Thanks @lb034582341!